### PR TITLE
Vietnamese form letters

### PR DESCRIPTION
**Pull request by timoballard**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/922)

## What does this change?
Adds form letter templates in Vietnamese. Migrations will need to be renumbered depending on the merge order for the form letter PRs. 

### TO DO:

- [x] Translate subject line static text
- [x] Translate & incorporate translated section names in form letter subject lines

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/models.py [hunk 1]
```diff
@@ -438,6 +444,11 @@ def available_report_fields(self, report):
                 'date_of_intake': format_date(report.create_date, format='long', locale='es_ES'),
                 'outgoing_date': format_date(today, locale='es_ES'),
                 'section_name': section_choices_es.get(report.assigned_section, "no section"),
+            },
+            'vi': {
+                'addressee': report.addressee_vi,
+                'date_of_intake': format_date(report.create_date, format='long', locale='vi'),
+                'outgoing_date': format_date(today, locale='vi'),
```
Comment (Jkrzy) [inline crt_portal/cts_forms/models.py:451]: We have `section_name` for es above, do we need it for vi?

File: crt_portal/cts_forms/migrations/0106_vietnamese_form_letters.py [hunk 2]
```diff
@@ -0,0 +1,675 @@
+from django.db import migrations
+
+def add_letters(apps, schema_editor):
+    ResponseTemplate = apps.get_model('cts_forms', 'ResponseTemplate')
+    subject = 'Response: Your Civil Rights Division Report - {{ record_locator }} from {{ section_name }} Section'
```
Comment (Jkrzy) [inline crt_portal/cts_forms/migrations/0106_vietnamese_form_letters.py:5]: Are we waiting on translations for the subject lines as well?

Comment (Jkrzy) [reply crt_portal/cts_forms/migrations/0106_vietnamese_form_letters.py:5]: Yes! You noted that in the PR, thanks Tim!

**Comment by timoballard** [comment]
@Jkrzy @ilodidoj - this is ready for review!

**Review by Jkrzy — APPROVED** [review]
Thanks @timoballard!

**Pull request merged**